onnxmltools/utils/float16_converter.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+from ..convert.common.data_types import *
+from onnxmltools.utils import load_model
+
+
+def npfloat16_to_int(list):
+    return [int(bin(_.view('H'))[2:].zfill(16), 2) for _ in list]
+
+
+def float16_converter(model_path):
+    """
+    Convert TensorProto type float to float16 in ONNX model.
+    :param model_path: ONNX model path and name
+    :return model: converted ONNX model
+
+    Example:
+
+    ::
+
+        from onnxmltools.utils import float16_converter
+        float16_converter('c:/winmlperf_coreml_SqueezeNet_prerelease.onnx')
+    """
+    model = load_model(model_path)
+    # create a queue for BFS
+    queue = []
+    if not model.graph: return None
+    queue.append(model)
+    while queue:
+        next_level = []
+        for q in queue:
+            # if q is model, push q.graph
+            try: q.graph
+            except: pass
+            else: next_level.append(q.graph)
```

Comment:
why not just if_else ?

Reply:
because q stored in the queue could be model, graph, or attributes. q.graph may not exist if q is not model.

Reply:
Couldn't we use if-else statement with `isinstance(q, ModelProto)`?